frontend/eda/rulebook-activations/RulebookActivationForm.tsx

```diff
@@ -138,8 +144,8 @@ export function RulebookActivationInputs() {
   const { data: tokens } = useGet<EdaResult<AwxToken>>(
     edaAPI`/users/me/awx-tokens/?page=1&page_size=200`
   );
-  const { data: webhooks } = useGet<EdaResult<EdaWebhook>>(edaAPI`/webhooks/?page=1&page_size=200`);
 
+  const [_, setDialog] = usePageDialog();
```

Comment:
We're in the process of deprecating `usePageDialog` in favor of `usePageDialogs`. The new hook is capable of handling use cases where we have stacked dialogs. Could you please replace `usePageDialog` with `usePageDialogs` here?